### PR TITLE
Fhir full url

### PR DESCRIPTION
**Pull request by evans-g-crsj**

### CODE REVIEW

File: config/output/config/dev-include.conf
Comment (evans-g-crsj) [inline config/output/config/dev-include.conf:1]: Generated

File: config/output/config/dev-kf-strides.conf
Comment (evans-g-crsj) [inline config/output/config/dev-kf-strides.conf:1]: Generated

File: config/output/config/prd-kf-strides.conf
Comment (evans-g-crsj) [inline config/output/config/prd-kf-strides.conf:1]: Generated

File: config/output/config/qa-include.conf
Comment (evans-g-crsj) [inline config/output/config/qa-include.conf:1]: Generated

File: config/output/config/qa-kf-strides.conf
Comment (evans-g-crsj) [inline config/output/config/qa-kf-strides.conf:1]: Generated

File: import-task/src/main/scala/bio/ferlab/fhir/etl/transformations/Transformations.scala [hunk 1]
```diff
@@ -195,6 +195,10 @@ object Transformations {
         .withColumnRenamed("docStatus", "status")
         .withColumn("relate_to", extractReferencesId(col("relatesTo.target.reference"))(0))
         .withColumn("latest_did", extractLatestDid(col("content")(0)("attachment")("url")))
+        .withColumn("fhir_document_reference",{
+          val fhirBaseUrl = split(col("full_url"), "/DocumentReference")(0)
```
Comment (jecos) [inline import-task/src/main/scala/bio/ferlab/fhir/etl/transformations/Transformations.scala:199]: 👍

**Review by jecos — APPROVED** [review]

**Review by jecos — APPROVED** [review]

**Review by jecos — APPROVED** [review]

**Pull request merged**